### DISCUSSION TITLE
Possible to autofill the currently selected completion?

### DISCUSSION THREAD

**Discussion by ddogfoodd** [saghen/blink.cmp#273] [Q&A] [answered]
Hey there,
is it possible to autofill the current completion selection?
Like whatever is selected in blink currently is pasted to where the cursor is so that it is not necessary to accept a selected item but instead you can just keep on typing whenever you selected the item you want.

I hope you get what I try to describe here.

**Comment by saghen** [comment, marked as answer]
Hey! Sounds like you're looking for `completion.list.selection = "auto_insert"` (>=0.7) or `windows.autocomplete.selection = 'auto_insert'` (<0.7)

**Reply by ddogfoodd** [reply]
Thank you! That's what I was looking for.
Just in case people come here in the future, there is a typo in your answer.

**Reply by saghen** [reply]
Fixed thanks :)

**Reply by rempas** [reply]
> Hey! Sounds like you're looking for `windows.autocomplete.selection = "auto_insert"`

The `selection` option is listed under `sources.list` in the [default configuration](https://github.com/Saghen/blink.cmp?tab=readme-ov-file#configuration) in the `README`. Please make sure no other mistakes are presented there. Thank you for your amazing work! :heart:

**Reply by saghen** [reply]
Yep there was a configuration rework so it moved. It's now under `completion.list.selection = "auto_insert"`, updated my original answer again

**Reply by rempas** [reply]
> Yep there was a configuration rework so it moved. It's now under `completion.list.selection = "auto_insert"`, updated my original answer again

Wait, I'm a little bit confused. For me, `completion.list.selection = "auto_insert` does NOT work but, `windows.autocomplete.selection = "auto_insert"` does...

**Reply by saghen** [reply]
Shit sorry, I should've clarified, the configuration rework has been merged on main but hasn't gone out in a release yet. It'll be out in 0.7+ for future readers

**Reply by rempas** [reply]
Thank you! Now, we'll know to change the config once the plugin stop working ;)
